File: 3818-minimum-prefix-removal-to-make-array-strictly-increasing/3818-minimum-prefix-removal-to-make-array-strictly-increasing.py
class Solution(object):
    def minimumPrefixLength(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums)
        for i in range(n - 1, 0, -1):
            if nums[i] <= nums[i - 1]:
                return i
        return 0

File: 3818-minimum-prefix-removal-to-make-array-strictly-increasing/3818-minimum-prefix-removal-to-make-array-strictly-increasing.cs
public class Solution {
    public int MinimumPrefixLength(int[] nums) {
        var n = nums.Length;
        for (var i = n - 1; i > 0; i--) {
            if (nums[i] <= nums[i - 1])
                return i;
        }
        return 0;
    }
}